charts/datahub/templates/elasticsearch-setup-job.yml

```diff
@@ -90,12 +90,7 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           resources:
-            limits:
-              cpu: 500m
```

Comment:
Could you make these parameters: like here: https://github.com/acryldata/datahub-helm/pull/223

Reply:
Would that be better than asking a user to define the entire `resource` block? What is the advantage here?
The block version is more verbose but also grants the most flexibility.